help/configure-unread-message-counters.md

```diff
@@ -0,0 +1,31 @@
+# Configure unread message counters
+
+To provide information about unread messages, Zulip can show unread message
+counters next to stream names in the left sidebar. This is likely to be helpful
+if you generally read (or otherwise [mark as read](/help/marking-messages-as-read))
+all the messages in a stream, but may otherwise feel unnecessary.
+
+To provide flexibility, Zulip offers an option to display a dot instead of an
+unread counter for streams in the left sidebar. You can use a dot for all
```

Comment:
Seeing it on the page, let's drop "To provide flexibility,"; it looks funny to start both paragraphs that way.